extensions/common/aws/aws-s3-core/src/main/java/org/eclipse/edc/aws/s3/S3CoreExtension.java

```diff
@@ -65,10 +67,13 @@ public AwsClientProvider awsClientProvider(ServiceExtensionContext context) {
 
         var threadPoolSize = context.getSetting(AWS_ASYNC_CLIENT_THREAD_POOL_SIZE, DEFAULT_AWS_ASYNC_CLIENT_THREAD_POOL_SIZE);
 
+        var pathStyleAccessEnabled = context.getSetting(AWS_PATH_STYLE_ACCESS_ENABLED, true);
```

Comment:
is `true` as default the behavior we want? currently isn't `false`?
In any case, please extract a constant, that will be used also in the `@Setting` annotation and in the configuration class.

Reply:
fixed the inconsistent default value and added constant.